fix: emphasize not to include generated theme file (#535)
vaadin/docs: articles/flow/styling/custom-theme-packaging.asciidoc

│                       └── my-theme  <1>
└── pom.xml
----

> .Do not include generated theme file
> [IMPORTANT]
> Do not include the automatically generated `<theme-name>.generated.js` file inside the packaged theme artifact.
> Including it can create errors for the project that tries to reuse the theme.
> The file is automatically created in the application project that uses the theme.

Reviewer: ```suggestion
.Do not include generated theme file
[IMPORTANT]
Do not include the automatically generated [filename]`<theme-name>.generated.js` file inside the packaged theme artifact.
The file is automatically created in an application project that uses the theme.
Including it in the theme can cause errors for such a project.
```
